test/jdk/java/lang/Thread/virtual/ThreadPollOnYield.java

```diff
@@ -41,14 +41,17 @@
  */
 
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.CountDownLatch;
 
 import jdk.test.lib.thread.VThreadPinner;
 import org.junit.jupiter.api.Test;
 import static org.junit.jupiter.api.Assertions.*;
 
 class ThreadPollOnYield {
+    public static CountDownLatch latch = new CountDownLatch(1);
     static void foo(AtomicBoolean done) {
         while (!done.get()) {
+            latch.countDown();
```

Comment:
While not correct, it looks very strange to do the count down in the loop. It would be a lot clearer to rename "latch" to "started" and do the countDown before the while.  Also no need for it to be public.

Reply:
Thanks. Variable has been renamed to `started`, and do the countDown before the while loop.